homeassistant/components/modbus/base_platform.py

```diff
@@ -92,11 +93,11 @@ async def async_update(self, now=None):
         """Virtual function to be overwritten."""
 
     @callback
-    def async_remote_start(self) -> None:
+    def async_run(self) -> None:
         """Remote start entity."""
-        if self._cancel_timer:
-            self._cancel_timer()
-            self._cancel_timer = None
+        self.async_hold(update=False)
+        if self._scan_interval == 0 | self._scan_interval > 2:
```

Comment:
Why is the interval `2` hardcoded here? Where does the number come from?

Reply:
hmmm you are right that should be a constant. Will make a change.

Reply:
Would also be nice, to maybe add some documentation to the constant. I see why `0` and `1` can be easily be special variables, but not sure which difference `2` and `3` makes semantically (as it does in your code, couldn't find an explanation for that).

Reply:
that is a general issue in this project, the number of comments are very close to zero, so that is practice. It is assumed that the code owner knows why. In this case there are really no difference between 2 or 3, but I had to pick a low number.

Reply:
Then maybe give a good example, comment the constant with "I had to pick a low random number", link a scientific source for such a number (https://xkcd.com/221/) and everything is fine :)

Nevermind. Thanks for considering.